### PR TITLE
Toggle active org

### PR DESCRIPTION
**Pull request by jay-meister**
expose function organisations.toggleActive

Which will turn the active key to the opposite of what it currently is.

Issue:
https://github.com/emfoundation/ce100-app/issues/418

### CODE REVIEW

**Comment by jay-meister** [comment]
@SimonLab @mk4111 any thoughts?

File: tests/orgs/toggleActive/endpoint.test.js
Comment (mk4111) [inline tests/orgs/toggleActive/endpoint.test.js]: Did you comment this out because we are still discussing a way to error handle

If that's the case, maybe it makes sense to remove this, write an issue to add a test later, that actually tests what it needs to? Because it's not doing anything at the moment...

**Review by mk4111 — APPROVED** [review]
Happy! Looks like it can almost be completely reused for peopleToggleActive too! Siiiick! :100:

**Comment by mk4111** [comment]
Just a bit of a pedantic point, but I'm wondering if it might be worth switching out the naming convention from 'ToggleActive' to 'ToggleEnable' (as per https://github.com/emfoundation/ce100-app/issues/376 ) so that we don't cause any further confusion with enable/disable and archive/unarchive? What are your thoughts @JMurphyWeb and @SimonLab ?

**Comment by jay-meister** [comment]
@mk4111 yeah I considered this too.

That is why I have changed it to toggleActive as opposed to the old version which was toggleArchive. 

In the DB we always have `active: `.  

On the front end we either have enable/disable or archive/unarchive. 

I think that now we know that enable/disable is for admins and the other is for users, then if we document it, it should be ok for now.

File: tests/orgs/toggleActive/endpoint.test.js
Comment (jay-meister) [reply tests/orgs/toggleActive/endpoint.test.js]: I will update this test when the query pr is merged

**Comment by jay-meister** [comment]
@SimonLab I will add that test back in once the query pr is merged. Please don't merge just yet 🙉

**Comment by jay-meister** [comment]
Updated.

**Comment by SimonLab** [comment]
@JMurphyWeb some merge conflict to resolved

**Comment by jay-meister** [comment]
ok

**Review by SimonLab — APPROVED** [review]
LGTM

**Pull request merged**
